e2e/deployers/deployers.go

```diff
@@ -34,9 +35,9 @@ func New(name string) (types.Deployer, error) {
 }
 
 func AvailableNames() []string {
-	// TODO: Use maps.Keys() when we have Go 1.23.
```

Comment:
When I added this comment I thought that map.Keys() returns []string but it returns `iter.Seq[K]` which is not helpful for our use case. If we want to iterate over the map keys iterating over the map is the best way.

The only way we can use map.Keys() is to return here iter.Seq[string] and let the caller create a []string, which does not help.

The best way to handle this comment is to remove it.

Reply:
Done

Reply:
Thanks, we have another comment like this in the workloads package. It would be good to remove it at the same time.

Sorry for not mention it in the previous review.

Reply:
Sure I will include that as well in this